tasks/main.yml

```diff
@@ -7,11 +7,17 @@
   when: docker_create_group
   tags:
     - config
-  
+
+- name: Create the docker group and add your user
+  command: usermod -aG docker {{ansible_ssh_user}}
```

Comment:
Should use proper `group` module for creating group. Can't avoid command for "adding" user to group though.
Use a variable for the user, not always the user making the connection that needs to be added to the group.
Group name can also be a variable if someone wants to use a different one.